### ISSUE TITLE
Not appearing to sync automatically 

### ISSUE THREAD

**Issue by shikhir-arora** [backstrokeapp/server#98]
Backstroke is working well - but I've noticed that all my repos - although the app says "Synced X minutes ago" (<10) -- doesn't actually seem to update. I have to manually push the resync button or manually queue the webhook from the URL.

**Comment by 1egoman** [comment]
I did a bit of investigation.

It looks like there was a typo in one of the api calls [here}(https://github.com/backstrokeapp/jobs/commit/8ade478f014767fe59ac996eb6b486b86f7a1494) that meant that the periodic checks that occur behind the scenes for each repository would actualy occur on the defualt branch for arepository, not the branch that was indicated in the link configuration. (I guess no one who was syncing to a different branch other than the default branch on the repository has complained before).

Anyway, here's a section of the row associated with one of your links, after I pushed the fix:
![2018-04-16-073511_1132x123_scrot](https://user-images.githubusercontent.com/1704236/38806909-d4ed5178-4148-11e8-8500-4f415d2ed1f6.png)

See the `upstreamLastSha` column? That now looks to properly reflect the HEAD of the rewrite branch: https://github.com/Rapptz/discord.py/commits/rewrite

Since I'm pretty confident that this fixed the issue, I'm going to close the issue, but feel free to ping me on this issue if I missed something.
